controllers/score-controllers.js

```diff
@@ -0,0 +1,58 @@
+const { Hospital, Score } = require('../db/models');
+const Sequelize = require('sequelize');
+
+exports.sendHospitalScore = function(req, res) {
+  const hospitalId = req.query.hospitalId;
+  const score = req.query.score;
+  const userid = req.query.userId;
+  Score.create({
+    rate: score,
+    UserId: userid,
+    HospitalId: hospitalId,
+  }).then(() => {
+    try {
+      const Op = Sequelize.Op;
+      Score.findAll({
+        where: {
+          HospitalId: hospitalId,
+          createdAt: {
+            [Op.lt]: new Date(),
+            [Op.gt]: new Date(new Date() - 15 * 60 * 1000),
+          },
+        },
+      }).then(scores => {
+        const readableScores = scores.map(score => score.toJSON());
```

Comment:
We could rename the variable to see better what it contains, what do you think?
```suggestion
        const lastScores = scores.map(score => score.toJSON());
```